app/views/podcasts/show.rss.builder

```diff
@@ -115,13 +115,20 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
           xml.itunes :keywords, ep.keywords.join(',') unless ep.keywords.blank?
           xml.itunes(:isClosedCaptioned, 'Yes') if ep.is_closed_captioned
 
-          xml.media(:content,
-            fileSize: ep.file_size,
-            type: ep.content_type,
-            url: ep.media_url) if ep.media?
-
           xml.content(:encoded) { xml.cdata!(ep.content) } unless ep.content.blank?
         end
+
+        if ep.media?
+          if @podcast.restrictions.present?
+            xml.media(:content, fileSize: ep.file_size, type: ep.content_type, url: ep.media_url) do
```

Comment:
Discovered that Spotify _does_ include non-full episodes in their app.  So they must fall back to the enclosure-url if the media:content is not present.

Had to move the `media:content` block(s) down here, so we can also include it for non-full episodes of a restricted podcast.

Reply:
Nevermind!  Should have been at `<item>` level.